### PR TITLE
Fix coveralls

### PR DESCRIPTION
**Pull request by ahartikainen**
Enable line-by-line info.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -72,6 +72,6 @@ script:
 after_success:
   - echo $(pwd)
   - ls -la
-  - if [[ $NAME == UNIT ]]; then head .coverage; fi
   - if [[ $NAME == SPHINX ]]; then travis-sphinx deploy; fi
-  - if [[ $NAME == UNIT ]]; then sed -i '/s/\/opt\/arviz/\/home\/travis\/build\/arviz-devs\/arviz/g' .coverage && coveralls; fi
+  - if [[ $NAME == UNIT ]]; then sed -i '/s/\/opt\/arviz/\/home\/travis\/build\/arviz-devs\/arviz/g' .coverage; fi
```
Comment (canyon289) [inline .travis.yml]: Love the bash magic :)

**Comment by ahartikainen** [comment]
Let's see if this works.

**Comment by canyon289** [comment]
LGTM

Example from branch
https://coveralls.io/jobs/43695633

File: .travis.yml [hunk 2]
```diff
@@ -40,7 +40,7 @@ install:
   - docker run --rm arviz:latest /bin/bash -c "conda info -a"
 
   # Print conda environment details
-  - docker run --rm arviz:latest /bin/bash -c "source /root/.bashrc && conda list"
+  - docker run --rm arviz:latest /bin/bash -c "source /root/.bashrc && conda list && pip freeze"
```
Comment (canyon289) [inline .travis.yml]: Good catch

**Pull request merged**